src/patchkernel/element.cpp

```diff
@@ -488,19 +488,24 @@ void Element::_initialize(long id, ElementType type, std::unique_ptr<long[]> &&c
 }
 
 /*!
-	Sets the ID of the element.
+	Gets the id that identifies the element.
```

Comment:
Should this comment say 'Set' ?

Reply:
Indeed. I will fix this before doing the merge. Thanks.